packages/emotion-theming/types/index.d.ts

```diff
@@ -20,3 +20,7 @@ export interface EmotionThemingModule<Theme> {
     ThemeProvider: ThemeProviderComponent<Theme>;
     withTheme<Props>(component: ComponentClass<Props> | SFC<Props>): ComponentClass<OptionalThemeProps<Props, Theme>>;
 }
+
+export const channel: string;
+
+export function createBroadcast(initialState: any): any;
```

Comment:
It does not return `any`. It clearly has `publish`, `subscribe`, `unsubscribe` as its properties.
And you could use generic type to remove `any` for `initialState` also.